src/controllers/krate/publish.rs

```diff
@@ -670,9 +670,9 @@ pub fn add_dependencies(
 impl From<TarballError> for BoxedAppError {
     fn from(error: TarballError) -> Self {
         match error {
-            TarballError::Malformed(err) => err.chain(cargo_err(
+            TarballError::Malformed(_err) => cargo_err(
```

Comment:
I'm assuming there's just no real value in the `TarballError` in practice, so it's not worth logging?

Reply:
The main cause of this variant is when the size limit is reached, but in that case it's a client error and we don't care that much about logging it since we don't need to act on it on our side